### ISSUE TITLE
ntfy attachments don't work

### ISSUE THREAD

**Issue by variasdesign** [YoRyan/mailrise#104]
Hello.

I'm trying to send an attachment to ntfy through mailrise. I'm using mailsend:

`mailsend -smtp mailrise.xyz -t log@mailrise.xyz -attach "/var/log/system.log" -sub "System log" -f me@example.org -M "Here's the system log:"`

However, the delivery fails:

`[2024-01-26 17:33:01] WARNING:mailrise.skeleton:Notification failed: address: [ me@example.org ➤ log@mailrise.xyz ] subject: [ System log ] body: [ Here's the system  (0.0K) ] attach: [ system.log (1536.8K) ]`

If I remove the attachment, everything works as expected. AFAIK, Apprise supports attachments in ntfy notifications. What could be wrong? Thanks for any help.

**Comment by AlBundy33** [comment]
try a smaller file.
afaik ntfy has a max message size.
maybe it's also an issue of the older apprise version (1.7.1) that is currently used.

**Comment by YoRyan** [comment]
Yeah, this is probably an attachment size limit. To be clear, Apprise does not communicate to Mailrise the reason for any delivery failure. It's a simple True/False success flag. There is also the logging information, if any.
